adapters/webhdfs-adapter/pom.xml

```diff
@@ -29,7 +29,7 @@
         <apache.httpcomponents.version>4.5.2</apache.httpcomponents.version>
         <apache.httpclient.version>4.5.12</apache.httpclient.version>
         <guava.version>20.0</guava.version>
-        <hadoop.version>2.7.3</hadoop.version>
+        <hadoop.version>2.10.1</hadoop.version>
```

Comment:
I'm not sure this will run with hadoop 2.10.x, we can only test it with a local cluster of the same version. In the itests, the mini hdfs cluster is used which has a different version of hadoop and some of the webHDFS calls are structured differently. This is fine because we won't be using the HDFS adapter, but I'm not sure it'll work as expected when switching versions if someone actually wanted to use it.

Reply:
I wasn't sure about the webhdfs stuff other than getting it to build and pass it's tests. I will remove it from the build again for now.